fileio/ft_read_data.m

```diff
@@ -1372,6 +1372,21 @@
       dat = read_yokogawa_data(filename, hdr, begsample, endsample, chanindx);
     end
     
+  case 'blackrock_nsx'
+    % use the NPMK toolbox for the file reading
+    ft_hastoolbox('NPMK', 1);
+    
+    % ensure that the filename contains a full path specification,
+    % otherwise the low-level function fails
+    [p,f,e] = fileparts(filename);
+    if ~isempty(p)
+      % this is OK
+    elseif isempty(p)
+      filename = which(filename);
+    end
+    orig = openNSx(filename, 'duration', [begsample endsample], 'channels', chanindx);
+    dat  = double(orig.Data);
```

Comment:
ft_read_data works fine with the current code if one continuous piece of recording is read in. However, this code will fail, if one stopped the recoding during the session and then started again. In this case,  orig.Data will be a {1xN cell} with N being the number of recording segments between start and end of recording. Any ideas how to deal with this?